presto-testng-services/src/main/java/io/prestosql/testng/services/ReportUnannotatedMethods.java

```diff
@@ -77,17 +76,33 @@ private static boolean isTestMethod(Method method)
         if (method.getDeclaringClass() == Object.class) {
             return true;
         }
-        Class<?> superclass = method.getDeclaringClass().getSuperclass();
-        Method overridden;
+        if (method.getDeclaringClass().isInterface()) {
+            return isTemptoClass(method.getDeclaringClass());
+        }
+        for (Class<?> interfaceClass : method.getDeclaringClass().getInterfaces()) {
+            if (isTemptoClass(interfaceClass)) {
+                return true;
```

Comment:
Does this disable ReportUnannotatedMethods for any class that happens to implement a tempto interface?